core/src/main/java/org/mule/routing/AbstractCorrelationAggregator.java

```diff
@@ -31,7 +31,7 @@ protected EventCorrelatorCallback getCorrelatorCallback(MuleContext muleContext)
     {
         public DelegateCorrelatorCallback(MuleContext muleContext)
         {
-            super(muleContext, persistentStores, storePrefix);
```

Comment:
It was used to create the store for the groups, now it is looked up from the registry after being put there by the EventCorrelator constructor.